apis/launchers.md

```diff
@@ -59,35 +59,43 @@ Granite.Services.Application.set_progress.begin (0.2f);
 
 As you can see, the method `set_progress` takes a `double` value type and is a range between `0` and `1`: from 0% to 100%. As with badges, Don't forget that `set_progress_visible` must be `true` and `.begin` is required for asynchronous methods.
 
-## Static Quicklists
+## Actions
 
-Static quicklists do not involve writing any code or using any external dependencies.
+Actions are specific functions your app can perform without already being open; think of them as alternate and more specific entry points into your app. Actions appear in the context menu of your app icon in the Applications Menu and Dock, and are searchable by name from the Applications Menu.
 
-Static quicklists are stored within your `.desktop` file. These are so called "actions". You can define many actions in your desktop file that will always show as an action in the application menu as well as in the dock.
+Adding actions to a `.desktop` file does not involve writing any code or using any external dependencies, though your app needs a way to distinguish between actions, e.g. with command line flags.
 
-The format is as follows:
+To add an action, add it by unique ID to a new `Actions` line in your app's .desktop file: 
 
-```text
+```ini
+Actions=ActionID;
+```
+
+Then below, add the action itself using the same unique ID:
+
+```ini
 [Desktop Action ActionID]
 Name=The name of the action
-Icon=The icon of the action (optional)
-Exec=The path to application executable and it's command line arguments (optional)
+Icon=An icon for the action (optional)
+Exec=application executable and any command line arguments
 ```
 
-Let's take a look at an example of an action that will open a new window of your application:
+The action name should not include your app's name, as it will always be displayed alongside your app. The action icon should also not be your app icon, as it may be shown in the menu for your app icon, or badged on top of the app icon.
```

Comment:
perhaps wrap this is a hint block?
```suggestion
{% hint style="info" %}
The action name should not include your app's name, as it will always be displayed alongside your app. The action icon should also not be your app icon, as it may be shown in the menu for your app icon, or badged on top of the app icon.
{% endhint %}
```